<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>792</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>852</width>
    <height>792</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>852</width>
    <height>792</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="m_XFieldscomboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X axis</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_YFieldscomboBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y axis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Plot">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Clear">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_Graph_holder" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>831</width>
      <height>671</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="widget_Graph"/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_PlotTypescomboBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_DataSet"/>
    <addaction name="actionExport_Grpah"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_DataSet">
   <property name="text">
    <string>Load DataSet</string>
   </property>
  </action>
  <action name="actionExport_Grpah">
   <property name="text">
    <string>Export Grpah</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
